### DISCUSSION TITLE
yt-dlp --sponsorblock-remove all

### DISCUSSION THREAD

**Discussion by notDavid** [Tzahi12345/YoutubeDL-Material#664] [Q&A] [answered]
Hi there,

is it possible to use the `--sponsorblock-remove all` option of `yt-dlp` when downloading videos?

Please advise, thank you :-)

**Comment by Tzahi12345** [comment, marked as answer]
Should be easy. First make sure you're using yt-dlp (you can set that in the `Advanced` tab in the settings)

Just go to the `Downloader` tab in the settings and in the `Global custom args` input, paste: `--sponsorblock-remove,,all`. The two commas are required to separate args. Hit save and try downloading a supported video, should work!

**Reply by notDavid** [reply]
Ahh yes indeed @Tzahi12345 it's working now, i forgot the double comma was needed 🤦🏼‍♂️

PS. the `Simulated command` says `youtube-dl` even when it's using `yt-dlp` which is a bit confusing 😅 I thought that was why it wasn't working.

Anyway problem solved, thank you!

**Reply by Tzahi12345** [reply]
Oh gotcha makes sense! I can update that in #657 so it shows the actual fork being used

**Reply by notDavid** [reply]
Great, thank you 👍🏼
